apps/backend/src/eventHandlers/MailService/SMTPMailService.ts

```diff
@@ -174,6 +174,22 @@ export class SMTPMailService extends MailService {
           id: 'isis-rapid-proposal-submitted-uo',
           name: 'ISIS Rapid User Office Submission Email',
         },
+        {
```

Comment:
Are there some emails missing here? I thought we would send the user an email when the proposal is `Approved` or `Unsuccessful` and possibly `Finished`?

Reply:
Updated